### ISSUE TITLE
Add support for CodePipeline

### ISSUE THREAD

**Issue by gholt-aws** [aws/aws-toolkit-vscode#711]
Support interacting with CodePipeline from VS Code

**Comment by bilalq** [comment]
Seems this just needs to do a lookup of pipelines within the CDK app and then call getPipelineState?

The presentation of this could go in either Explorer or Developer Tools, but I think the latter would probably be a better fit. 

One other interesting concern here is cross-account pipelines. Since the toolkit extension only loads a single profile at a time, users have to choose between loading either an account where a stage is deployed or the pipeline itself. There's no way to actually see the broader picture. That probably merits its own issue to discuss.

**Comment by justinmk3** [comment]
> One other interesting concern here is cross-account pipelines. Since the toolkit extension only loads a single profile at a time, users have to choose between loading either an account where a stage is deployed or the pipeline itself. There's no way to actually see the broader picture.

Great point. We are thinking about cross-account and multi-account support, but it's very early stages.

**Comment by hbracy** [comment]
I would love this :)

**Comment by andrew-dixon** [comment]
I was surprised this actually wasn't already one of the AWS tools included, along with CodeBuild as well. I think adding both CodePipeline and CodeBuild would be excellent additions.

**Comment by agalazis** [comment]
I don't think it really needs cdk, we could just add a pipeline arn in vscode workspace settings which can be picked

**Comment by karthikghantasala** [comment]
This will be an awesome feature to have.. and this shouldnt need anything to do with CDK
